server/lib/settings/index.ts

```diff
@@ -547,6 +560,15 @@ class Settings {
         csrfProtection: false,
         forceIpv4First: false,
         trustProxy: false,
+        trustedProxies: {
+          v4: [],
+          v6: [],
+        },
```

Comment:
if forward auth is going to take a while, can we have the trusted proxies list merged in separate? as it is currently, enabling "trustProxy" has security implications as requests can be forged, unless a firewall prevents accessing the instance directly into the 5055 port.